radis/lbl/broadening.py

```diff
@@ -2203,14 +2207,37 @@ def _broaden_lines(self, df):
                         abscoeff += absorption
                         pb.update(i)
                     pb.done()
+                elif optimization in ("simple", "min-RMS"):
+                    self.reftracker.add(doi["DIT-2020"], "algorithm")
+                    for i, (_, dg) in enumerate(df.groupby(arange(len(df)) % N)):
+                        (
+                            line_profile_LDM,
+                            wL,
+                            wG,
+                            wL_dat,
+                            wG_dat,
+                        ) = self._calc_lineshape_LDM(dg)
+                        (wavenumber, absorption) = self._apply_lineshape_LDM(
+                            df.S.values,
```

Comment:
shouldn't you use `dg.S.values` here?

Reply:
Yes you're right, I actually changed that a while back. I'll commit the latest version soon.